### DISCUSSION TITLE
Open a new child tab right below the parent one.

### DISCUSSION THREAD

**Discussion by pafke2** [piroor/treestyletab#3076] [General]
Good day, is there a way when you have a parent tab with child tabs and you create another child tab it gets placed right below the parent tab? Because right now the new child tab gets placed at the bottom - below the previously opened child tabs.
The scenario is:
I have a parent tab (let's call it A) with one child tab (B), and I open another child tab (C), the order will be:
1 A
2 - B
3 - C.
And I want the C (and all the newest child tabs) to be placed higher (2) than the previous ones.
1 A
2 - C
3 - B

**Comment by irvinm** [comment]
Try playing around with these options to see if they fit your needs:

![image](https://user-images.githubusercontent.com/979729/153686986-348a760a-ed12-40e8-8bb5-3711eba2146c.png)
